release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,54 @@
-
-[id="supported-plugins-in-rhdh"]
-.Supported plugins in {product} {product-version}
-[cols="30%,30%,10%,30%",options="header",subs="+attributes"]
-|===
-| Plugin name | Plugin ID | Version | Category
-
-| Segment | `@janus-idp/backstage-plugin-analytics-provider-segment` | 1.2.0 | Analytics
-
-|Keycloak |`@janus-idp/backstage-plugin-keycloak-backend` |1.7.6 |Identity and access management
-
-|Open Cluster Management (OCM)|`@janus-idp/Backstage-plugin-ocm`|3.5.0|Cluster management
-
-|OCM backend|`@janus-idp/backstage-plugin-ocm-backend`|3.4.6|Cluster management
-
-|Quay|`@janus-idp/backstage-plugin-quay`|1.4.6|Container management
-
-|Quay actions|`@janus-idp/backstage-scaffolder-backend-module-quay`|1.2.1|Container management
-
-|RBAC backend|`@janus-idp/backstage-plugin-rbac-backend`|1.6.3|Security
-
-|RBAC common|`@janus-idp/backstage-plugin-rbac-common`|1.1.0|Security
-
-|Tekton|`@janus-idp/backstage-plugin-tekton`|3.1.3|Cluster management
-
-|Topology|`@janus-idp/backstage-plugin-topology`|1.16.4|Cluster management
-
-|Regex actions|`@janus-idp/backstage-scaffolder-backend-module-regex`|1.2.1|Continuous deployment
-
-|ArgoCD|`@roadiehq/backstage-plugin-argo-cd`|2.3.5|Continuous deployment
-
-|ArgoCD backend|`@roadiehq/backstage-plugin-argo-cd-backend`|2.11.3|Continuous deployment
-
-|API Documentation|`@backstage/plugin-api-docs`|0.9.12|Continuous deployment
-
-|App backend|`@backstage/plugin-app-backend`|0.3.54|Continuous deployment
-
-|Auth backend|`@backstage/plugin-auth-backend`|0.19.4|Security
-
-|Auth node|`@backstage/plugin-auth-node`|0.4.0|Security
-
-|Catalog frontend|`@backstage/plugin-catalog`|1.14.0|Continuous deployment
-
-|Catalog backend|`@backstage/plugin-catalog-backend`|1.14.0|Continuous deployment
+=== Configuration change in the GitHub Organization catalog provider  
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings.
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
+=== New Scaffolder Identity client
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+Identity client functioning differs slightly between the one provided by the new backend system Scaffolder plugin and the one used in the previous release:
 
-|Catalog React|`@backstage/plugin-catalog-react`|1.8.5|Continuous deployment
+* The previous release used the `DefaultIdentityClient` function without any specific option
+* The new backend system Scaffolder does not use the default identity service, but still uses the deprecated scaffolder-dedicated `buildDefaultIdentityClient` function, which introduces a slight functioning change in the identity client. 
```

Comment:
It seems to me that your proposal sounds correct. As for for it means in more details for the enduser experience, maybe @kadel can answer better that me since he tested it.